pkg/agent/apiserver/apiserver.go

```diff
@@ -138,6 +138,9 @@ func newConfig(npq querier.AgentNetworkPolicyInfoQuerier, bindPort int, enableMe
 	if err := secureServing.ApplyTo(&serverConfig.SecureServing, &serverConfig.LoopbackClientConfig); err != nil {
 		return nil, err
 	}
+	// TODO: unable to load configmap based request-header-client-ca-file
+	// 1. configure somewhere here to input user specific k8s service ip:port
+	// 2. re-order the startup process to have antreaproxy started first, then new apiserver on agent.
```

Comment:
is it just a transient error and we can eventually serve APIs (once AntreaProxy is running)? In other words, are you just trying to avoid an error message in the agent logs?

Reply:
Sorry, these comments should have been deleted. It is just a reminder for myself at that time...

When I finished the code (endpoint hostnetwork is false), I met the `unable to load configmap based request-header-client-ca-file` error. The solution is to support hostnetwork is true (k8s service), which is implemented in this PR.

About the startup order, I see that in TOT code, antrea-proxy is already before `apiserver.New()`. Second TODO is actually unnecessary.